device/arista/x86_64-arista_7050cx3_32s/Arista-7050CX3-32S-C28S4/config.bcm.j2

```diff
@@ -0,0 +1,543 @@
+{# Construct config.bcm to include additional soc properties per specific device metadata requirement #}
```

Comment:
Is it possible to use the C32 BCM file to start with? Although MMU settings might be different for optimized config, but it should not matter that much.

Reply:
Confirmed with BCM. The buffer config is not required in this file.